### PR TITLE
adds website table to schema and insert.sql

### LINKED ISSUE #79: Update ERD to work with multiple websites
The ERD currently has the support for multiple websites, but being more explicit by adding a new table could help move things along:

**Todo**
- [x] Confer with team about ERD diagram
- [x] Create a new issue and pull request to update the Schema.SQL
- [x] Create issues pull requests to update the previous sections to work with this new schema.

### PR DESCRIPTION
**Pull request by danfh00**
Adds website table to schema and example data added to insert.sql for potential websites. closes #79

### CODE REVIEW

File: Database/insert.sql [hunk 1]
```diff
@@ -1,26 +1,25 @@
...
+INSERT INTO products (product_id, url, product_code, product_name, website_id) OVERRIDING SYSTEM VALUE VALUES
+(1, 'https://www.asos.com/adidas-performance/adidas-running-response-trainers-in-white-and-blue/prd/203474246#colourWayId-203474304', '203474246', 'adidas Running Response trainers in white and blue', 1),
+(2, 'https://www.asos.com/new-balance/new-balance-hierro-mid-trail-running-trainers-in-black/prd/202421541#colourWayId-202421546', '202421541', 'New Balance Hierro Mid trail running trainers in black', 1);
 
 -- Inserting example data into the users table
-INSERT INTO users (user_id, email) OVERRIDING SYSTEM VALUE VALUES
-(1, 'user1@example.com'),
-(2, 'user2@example.com'),
-(3, 'user3@example.com');
+INSERT INTO users (email) OVERRIDING SYSTEM VALUE VALUES
```
Comment (BerkayDur) [inline Database/insert.sql]: Do we need to reset index?

File: Database/schema.sql [hunk 2]
```diff
@@ -1,11 +1,16 @@
-DROP TABLE IF EXISTS price_readings, subscriptions, users, products;
+DROP TABLE IF EXISTS websites, price_readings, subscriptions, users, products;
 
+CREATE TABLE websites (
+    website_id INTEGER GENERATED ALWAYS AS IDENTITY PRIMARY KEY,
```
Comment (BerkayDur) [inline Database/schema.sql]: I think our ERD diagram uses smallint

**Review by BerkayDur — CHANGES_REQUESTED** [review]
A few suggestions. Feel free to ignore them

File: Database/insert.sql (hunk 1 above)
Comment (danfh00) [reply Database/insert.sql]: No, removed this

File: Database/schema.sql (hunk 2 above)
Comment (danfh00) [reply Database/schema.sql]: Thanks, updated

File: Database/schema.sql [hunk 3]
```diff
@@ -1,11 +1,16 @@
...
+    website_name TEXT UNIQUE NOT NULL
+);
 
 CREATE TABLE products(
     product_id INTEGER GENERATED ALWAYS AS IDENTITY PRIMARY KEY,
     url TEXT UNIQUE NOT NULL,
     product_code TEXT NOT NULL,
-    product_name TEXT NOT NULL
+    product_name TEXT NOT NULL,
+    website_id INTEGER NOT NULL REFERENCES websites (website_id)
```
Comment (BerkayDur) [inline Database/schema.sql]:
```suggestion
    website_id SMALLINT NOT NULL REFERENCES websites (website_id)
```

**Review by BerkayDur — CHANGES_REQUESTED** [review]

**Review by A-Faris — DISMISSED** [review]
Database looks good

**Review by BerkayDur — APPROVED** [review]
Other than the smallint, it looks good!

**Pull request merged**